edb/lib/std/25-numoperators.eql

```diff
@@ -930,23 +1034,43 @@ CREATE INFIX OPERATOR
 std::`%` (n: std::decimal, d: std::decimal) -> std::decimal
 {
     FROM SQL OPERATOR r'%';
-    FROM SQL 'SELECT n - floor(n / d) * d';
+    FROM SQL 'SELECT n - floor("n" / "d") * d';
 };
 
 
+# need an explicit operator for int64 in order to guarantee the result
+# is float64 and not decimal
 CREATE INFIX OPERATOR
-std::`^` (n: std::float64, p: std::float64) -> std::float64
-    FROM SQL OPERATOR '^';
+std::`^` (n: std::int64, p: std::int64) -> std::float64
+{
+    # We cheat here a bit by copying the `/` operator, since there is
```

Comment:
"by copying the `/` operator" -> "by copying most of SQL operator metadata from the `/` SQL operator"

Reply:
Replaced this comment with a slightly longer more explicit version in this file.